gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/ApiReactorHandlerFactory.java

```diff
@@ -90,25 +90,29 @@ public class ApiReactorHandlerFactory implements ReactorHandlerFactory<Api> {
     public static final String REPORTERS_LOGGING_MAX_SIZE_PROPERTY = "reporters.logging.max_size";
     public static final String HANDLERS_REQUEST_HEADERS_X_FORWARDED_PREFIX_PROPERTY = "handlers.request.headers.x-forwarded-prefix";
     public static final String REPORTERS_LOGGING_EXCLUDED_RESPONSE_TYPES_PROPERTY = "reporters.logging.excluded_response_types";
-    private static final String PENDING_REQUESTS_TIMEOUT_PROPERTY = "api.pending_requests_timeout";
     public static final String API_JUPITER_MODE_ENABLED = "api.jupiterMode.enabled";
+    private static final String PENDING_REQUESTS_TIMEOUT_PROPERTY = "api.pending_requests_timeout";
```

Comment:
to stay consistent with previous constant
```suggestion
    private static final String PENDING_REQUESTS_TIMEOUT = "api.pending_requests_timeout";
```

Reply:
I am not the one that create this, the formatter move it because it is private but actually all others have PROPERTY suffix except jupiter mode. I will add property to jupiter mode
![image](https://user-images.githubusercontent.com/4324887/174977053-4f9c9260-bf70-409c-80ed-d84328b386b2.png)